src/payment/strategies/stripev3/stripev3-payment-strategy.ts

```diff
@@ -0,0 +1,123 @@
+        const stripeOptions = options.stripev3;
+
+        if (!stripeOptions) {
+            throw new InvalidArgumentError('Unable to initialize payment because "options.stripev3" argument is not provided.');
+        }
+
+        const paymentMethod = this._store.getState().paymentMethods.getPaymentMethod(options.methodId);
+
+        if (!paymentMethod) {
+            throw new MissingDataError(MissingDataErrorType.MissingPaymentMethod);
+        }
+
+        return this._stripeScriptLoader.load(paymentMethod.initializationData.stripePublishableKey)
+            .then(stripeJs => {
+                this._stripeV3Client = stripeJs;
+                const elements = this._stripeV3Client.elements();
+                const cardElement = elements.create('card', {
+                    style: stripeOptions.style,
+                });
+
+                cardElement.mount(`#${stripeOptions.containerId}`);
+
+                this._cardElement = cardElement;
+
+                return Promise.resolve(this._store.getState());
+            });
+    }
+
+    execute(payload: OrderRequestBody, options?: PaymentRequestOptions): Promise<InternalCheckoutSelectors> {
+        const { payment, ...order } = payload;
+
+        if (!payment) {
+            throw new PaymentArgumentInvalidError(['payment']);
+        }
+
+        return this._store.dispatch(this._paymentMethodActionCreator.loadPaymentMethod(payment.methodId))
+            .then(state => {
+                const paymentMethod = state.paymentMethods.getPaymentMethod(payment.methodId);
+
+                if (!paymentMethod || !paymentMethod.clientToken) {
+                    throw new MissingDataError(MissingDataErrorType.MissingPaymentMethod);
+                }
+
+                return this._getStripeJs().handleCardPayment(
+                    paymentMethod.clientToken, this._getCardElement(), {}
+                ).then(stripeResponse => {
+                    if (stripeResponse.error || !stripeResponse.paymentIntent.id) {
+                        throw new StandardError(stripeResponse.error && stripeResponse.error.message);
+                    }
+
+                    const paymentPayload = {
+                        methodId: payment.methodId,
+                        paymentData: { nonce: stripeResponse.paymentIntent.id },
+                    };
+
+                    return this._store.dispatch(this._orderActionCreator.submitOrder(order, options))
+                        .then(() =>
+                            this._store.dispatch(this._paymentActionCreator.submitPayment(paymentPayload))
+                        );
+                });
+            });
+    }
+
+    finalize(options?: PaymentRequestOptions): Promise<InternalCheckoutSelectors> {
+        return Promise.reject(new OrderFinalizationNotRequiredError());
+    }
+
+    deinitialize(options?: PaymentRequestOptions): Promise<InternalCheckoutSelectors> {
+        this._getCardElement().unmount();
```

Comment:
You should catch `NotInitializedError` because `unmount` action is actually not critical so we shouldn't halt the execution of the app. That again is one of the reasons why we want you guys to do an explicit check instead of using convenient getters, because here you can just return early instead of having to do a `try/catch`.